main/trello/api/lists_manager.py

```diff
@@ -0,0 +1,60 @@
+from urllib import response
+from main.trello.api.rest_base_manager import RESTBaseManager
+
+"""
+Module to manage Trello Lists through the API
+
+Classes:
+    ListsManager
+"""
+class ListsManager(RESTBaseManager):
+    """Class which can be used to manage Trello Lists through the API"""
+
+    def __init__(self, request_method=None):
+        super().__init__(request_method)
+        """ Construct the necessary attributes for the ListsManager
+
+        :param method: obj  RequestManager object which is used to handle the API requests
+        """
```

Comment:
The doc string needs to be right below the signature:
```suggestion
        """ Construct the necessary attributes for the ListsManager
        :param method: obj  RequestManager object which is used to handle the API requests
        """
        super().__init__(request_method)
```